exercises/perfect-numbers/example.js

```diff
@@ -1,64 +1,49 @@
-
-export default class PerfectNumbers {
-
-  /**
-   * Calculate all the divisors for a given number.
-   * @param {number} number - natural number.
-   * @returns {array} array that contains the divisors for a given number NOT including the number itself.
-   */
-  getDivisors(number) {
-    let i;
-    const divs = new Array();
-
-    // Accepts only natural numbers greater than 1.
-    if (number <= 1) {
-      return divs;
-    }
-
-    // 1 always divides everyone!
-    divs.push(1);
-
-    // Calculate the divisors up the the half of the number + 1
-    for (i = 2; i <= number / 2; i++) {
-      if (number % i === 0) {
-        divs.push(i);
-      }
-    }
-
+/**
+  * Calculate all the divisors for a given number.
+  * @param {number} number - natural number.
+  * @returns {array} that contains the divisors for a given number excluding the number itself.
+  */
+function getDivisors(number) {
```

Comment:
I like the way you've set this method up!  At first I was puzzled why you weren't returning `[1]` for every case, but now I see what you're doing.